src/configGenerator/configLocator.ts

```diff
@@ -21,6 +21,7 @@ export async function findConfigFrom(path: string) {
   );
   assert(await isDir(path), '`path` must be a directory');
 
+  let oldPath = null;
```

Comment:
Nit:

```suggestion
  let parentPath = null;
```

Reply:
oldPath doesn't contain the parent. It's either a child of currentPath or the same as currentPath. I think the name "prevPath" is probably clearer, as it should contain the the path from the previous iteration of the loop.

Reply:
I have made this change. PTAL